docs/_blog/cashapp-enables-lightning-withdrawals-and-deposits-using-ldk.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "CashApp enables lightning withdrawals & deposits using LDK"
+description: "Learn how the CashApp team integrated with the Lightning Network, allowing its users to send and receive bitcoin instantly."
+date: "2022-11-01"
+authors:
+  - Ryan Loomba
+tags:
+  - Case Studies 
+--- 
+
+# Overview
+Our project is the Cash App lightning wallet. We power all lightning bitcoin withdrawals and deposits within Cash App.
```

Comment:
This first sentence sounds funny. I think you can just leave out "our project", but need to think about it.